qiskit/algorithms/evolvers/real/trotterization/trotter_qrte.py

```diff
@@ -0,0 +1,213 @@
+
+            from qiskit.opflow import X, Y, Zero
+            from qiskit.algorithms import EvolutionProblem, EvolutionResult
+            from qiskit.algorithms.evolvers.real.implementations.\
+                trotterization.trotter_qrte import TrotterQrte
+
+            operator = X + Z
+            initial_state = Zero
+            time = 1
+            evolution_problem = EvolutionProblem(operator, 1, initial_state)
+            # LieTrotter with 1 rep
+            trotter_qrte = TrotterQrte(evolution_problem)
+            evolved_state = trotter_qrte.evolve().evolved_state
+    """
+
+    def __init__(
+        self,
+        product_formula: ProductFormula = None,
+        expectation: Optional[ExpectationBase] = None,
+        quantum_instance: Union[QuantumInstance, BaseBackend, Backend] = None,
+    ) -> None:
+        """
+        Args:
+            product_formula: A Lie-Trotter-Suzuki product formula. The default is the Lie-Trotter
+                first order product formula with a single repetition.
+            expectation: An instance of ExpectationBase which defines a method for calculating
+                expectation values of EvolutionProblem.aux_operators.
+            quantum_instance: A quantum instance used for calculations.
+        """
+        if product_formula is None:
+            product_formula = LieTrotter()
+        self._product_formula = product_formula
+        self._quantum_instance = quantum_instance
+        self._expectation = expectation
+
+        self._circuit_sampler = CircuitSampler(quantum_instance)
+
+    @property
+    def product_formula(self) -> ProductFormula:
+        """Returns a product formula used in the algorithm."""
+        return self._product_formula
+
+    @property
+    def quantum_instance(self) -> Union[QuantumInstance, BaseBackend, Backend]:
+        """Returns a quantum instance used in the algorithm."""
+        return self._quantum_instance
+
+    @quantum_instance.setter
+    def quantum_instance(self, quantum_instance: Union[QuantumInstance, BaseBackend, Backend]):
+        """Sets a quantum instance used in the algorithm."""
+        self._quantum_instance = quantum_instance
+
+    @property
+    def expectation(self) -> ExpectationBase:
+        """Returns an expectation used in the algorithm."""
+        return self._expectation
+
+    @classmethod
+    def supports_aux_operators(cls) -> bool:
+        """
+        Whether computing the expectation value of auxiliary operators is supported.
+
+        Returns:
+            True if `aux_operators` expectations in the EvolutionProblem can be evaluated, False
+                otherwise.
+        """
+        return True
+
+    def evolve(self, evolution_problem: EvolutionProblem) -> EvolutionResult:
+        """
+        Evolves a quantum state for a given time using the Trotterization method
+        based on a product formula provided.
+
+        Note: Time-dependent Hamiltonians are not yet supported.
```

Comment:
If you want, you could use the Sphinx command for notes here, that should put it in a highlighted box in the docs
```suggestion
        .. note::
        
            Time-dependent Hamiltonians are not yet supported.
```